src/Routes/ImageManager/Images.js

```diff
@@ -81,6 +100,106 @@ const Images = () => {
     }),
     shallowEqual
   );
+
+  useEffect(() => {
+    if (data) {
+      setRows(
+        flatten(
+          data.map(item => {
+            const statusIconHash = {
+              "Ready": <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
```

Comment:
Right now focus only on "BUILDING". I am not sure what the status will be on Ready and Building error.